helm/metac/values.yaml

```diff
@@ -0,0 +1,52 @@
+# Default values for metac.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+## Metac image to use
+##
+image:
+  repository: quay.io/amitkumardas/metac
+  tag: latest
```

Comment:
Can we have a proper tag instead of `latest` . apparently this can be changed once we have a release created.

Reply:
There is no proper tag that includes the flags that we're using to start metac, only `latest` and `master` have those flags....

Reply:
ok... we can take this up once we have the next release